### ISSUE TITLE
[Question] Regarding latest build on curseforge

### ISSUE THREAD

**Issue by ChromesDuzez** [Chocohead/OptiFabric#1138]
Does optifabric-1.13.25.jar build of optifabric include this pull? #1110 [Fix Carpet 1.19.3-1.20.1](https://github.com/Chocohead/OptiFabric/pull/1110)

**Comment by topia** [comment]
No. All public releases were released from merged branches, eg. `llama`, not any PRs.

**Comment by ChromesDuzez** [comment]
ok what is the best way for me to build the codebase with that PR included? I've never built a java project for mc before

**Comment by topia** [comment]
It's a simple gradle project to build. By the way, you might want to check the build artifact from  https://github.com/Chocohead/OptiFabric/actions/runs/5589052799 first.

edit: correct link

**Comment by ChromesDuzez** [comment]
~~looking through the log of that build, looks like a lot of renderer warnings and other related warnings has anyone tried running a build of the project in that state? I'm worried I'm going to take the time to download the project, build then find out it just doesn't work lol~~

 I see the build artifacts was very helpful... and it did work for me nice! tysm

**Comment by topia** [comment]
Nice. Could you close the issue if your problem is solved?
